acceptance/suites/tests/00_smoke/puppetdb_integration.rb

```diff
@@ -0,0 +1,52 @@
+## Tests that PuppetDB can be integrated with Puppet Server in a simple
+## monolithic install (i.e. PuppetDB on same node as Puppet Server).
+##
+## In this context 'integrated' just means that Puppet Server is able to
+## communicate over HTTP/S with PuppetDB to send it information, such as
+## agent run reports.
+##
+## This test validates communication is successful by querying the PuppetDB HTTP
+## API and asserting the agent's report timestamp is not null. This means that
+## PuppetDB successfully received the agent's report sent from Puppet Server.
+## We can just run the agent that's on the master for this.
+
+confine :to, {:platform => puppetdb_supported_platforms}, master
+
+require 'json'
+
+test_name 'PuppetDB integration'
+
+step 'Install PuppetDB module' do
+  on(master, puppet("module install puppetlabs-puppetdb --version 5.0.0"))
```

Comment:
We could, yeah.  Or we could unpin it and see how far our luck goes?  I didn't want to spread this code out too much in the first attempt, but if we think it's worthwhile to expose a bit more I'd be happy to.